Lib/test/test_syntax.py

```diff
@@ -461,22 +461,22 @@
 >>> def foo(a,*b=3,c):
 ...    pass
 Traceback (most recent call last):
-SyntaxError: var-positional parameter cannot have default value
+SyntaxError: var-positional parameter's default value cannot be changed because it has the immutable default value ()
 
 >>> def foo(a,*b: int=,c):
 ...    pass
 Traceback (most recent call last):
-SyntaxError: var-positional parameter cannot have default value
+SyntaxError: var-positional parameter's default value cannot be changed because it has the immutable default value ()
 
 >>> def foo(a,**b=3):
 ...    pass
 Traceback (most recent call last):
-SyntaxError: var-keyword parameter cannot have default value
+SyntaxError: var-keyword parameter's default value cannot be changed because it has the immutable default value {}
```

Comment:
Same as above, but this is worse -- it confuses the reader over `{}` (which is not immutable) vs the default (which notionally is).